test/D2L.Security.OAuth2.Benchmarks/FullStackValidation/FullStackValidationBenchmarks.cs

```diff
@@ -1,27 +1,49 @@
 ﻿using System;
-using BenchmarkIt;
+using System.Collections.Generic;
+using BenchmarkDotNet.Attributes;

```

Comment:
For .NET Standard?

Reply:
Like why switch? Yeah I think so?

The commit is almost 2 years old. https://github.com/Brightspace/D2L.Security.OAuth2/commit/9de36e5db5d352ec46bfe4ede476f8ec3d42b7c1

Reply:
Haha